getting-started/setup-building.rst

```diff
@@ -385,7 +385,7 @@ their dependencies::
          lzma lzma-dev tk-dev uuid-dev zlib1g-dev
 
 
-.. _MacOS:
+.. _macOS:
```

Comment:
Can we keep both the old and the new anchor for potentially existing hyperlinks? AFAIK URL paths and anchors are case sensitive.